### DISCUSSION TITLE
GIMP Scripts for Packing Textures

### DISCUSSION THREAD

**Discussion by brohd11** [TokisanGames/Terrain3D#377] [Extras]
Hey guys, not sure where to post this, but made some Python scripts for channel packing the textures in GIMP. It will decompose, recompose and then export to dds in 1 click.

Hope it helps someone!

https://github.com/brohd11/Texture-Packing-Scripts-for-GIMP

**Comment by TokisanGames** [comment]
Great thanks. There's a channel packer in the Terrain3D tools menu, but I still use Gimp for other channel editing.
